Tasks/ManualIntervention/task.json

```diff
@@ -3,7 +3,7 @@
     "name": "ManualIntervention",
     "friendlyName": "Manual Intervention",
     "description": "Pause deployment and wait for intervention",
-    "helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=825235)",
+    "helpMarkDown": "[More Information](https://docs.microsoft.com/en-us/vsts/build-release/tasks/utility/manual-intervention)",
```

Comment:
We should use only forward link so that it can take you to language specific help link. No?